pydrad/parse/util.py

```diff
@@ -90,28 +90,32 @@ def read_amr_file(filename):
         'electron_energy_density': 'erg cm-3',
         'hydrogen_energy_density': 'erg cm-3',
     }
-    table = astropy.table.QTable.read(
+    # NOTE: Using pandas here because it is much faster than using the
+    # I/O functionality in astropy for these types of tables
+    table = read_csv(
         filename,
-        format='ascii',
-        data_start=4,
+        skiprows=4,
+        sep=r'\s+',
+        header=None,
+        engine='c',
     )
     # NOTE: The columns we care about are doubles in HYDRAD, while the
     # other columns are integers with information about the
-    # refinement level of the grid cell.  As a result, if electron
+    # refinement level of the grid cell. As a result, if electron
```

Comment:
lol.  The old two spaces after a period debate!  😃

Reply:
No comment 😆 🙈